src/test/resources/com/puppycrawl/tools/checkstyle/checks/design/finalclass/InputFinalClass.java

```diff
@@ -121,13 +121,13 @@ private someinnerClass() {}
 }
 
 interface TestInterface {
-    class SomeClass {
+    final class SomeClass { // ok
```

Comment:
Why change test case here?

Reply:
This test case (and the next one) was supposed to prove that this check does not check internal classes for interfaces.
They are no longer relevant. I changed them both.

Reply:
Resolved.

Reply:
can we remove "final" from this line and place a comment that Check skip them

Reply:
done